corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -524,6 +528,8 @@
         {% include 'hqwebapp/partials/ko_search_box.html' %}
         {% include 'hqwebapp/partials/ko_select_toggle.html' %}
 
+        <script src="{% static 'hqwebapp/js/downgrade_modal.js' %}"></script>
```

Comment:
This JS file would now be requested on each page load even if it is not needed. I believe you should be able to add
`{% if show_overdue_invoice_modal %}` here? or add it in the modal itself.

Reply:
Updated